### PR TITLE
RHDEVDOCS-4926- Logging 5.4.11 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9+

Issue: [RHDEVDOCS-4926](https://issues.redhat.com//browse/RHDEVDOCS-4926)

Link to docs preview: https://55637--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-4-11_cluster-logging-release-notes-v5x

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by gabriel-rh — APPROVED** [review]
/lgtm

File: modules/cluster-logging-rn-5.4.11.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//module included in cluster-logging-release-notes.adoc
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-rn-5.4.11.adoc:1]:
```suggestion
// Module included in the following assemblies:
//
// * logging/cluster-logging-release-notes.adoc


```
**Suggestion**: Refer to the guideline and example [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata).

File: modules/cluster-logging-rn-5.4.11.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-11_{context}"]
+= Logging 5.4.11
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0632[OpenShift Logging Bug Fix Release 5.4.11].
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-rn-5.4.11.adoc:5]: Getting 404 PAGE NOT FOUND for the errata link. Hope it is correct. Please check.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@libander Nice work! PTAL at the suggestions. Thanks!

**Pull request merged**